src/libZilliqa/Zilliqa.cpp

```diff
@@ -336,6 +337,10 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
       m_lookupServer =
           make_shared<LookupServer>(m_mediator, *m_lookupServerConnector);
 
+      if (ENABLE_WEBSOCKET) {
```

Comment:
I think in a singleton , it gets initialized the first time you call it. The statement seems unneeded

Reply:
I think we need it. If the server `start` fails we reset the ENABLE_WEBSOCKET.
If we rely on getinstance during calling functions, it will be too late otherwise.